docs/production/deployment.md

```diff
@@ -223,28 +223,14 @@ behind reverse proxies.
 
 [using-http]: ../production/deployment.html#configuring-zulip-to-allow-http
 
-## Using an outgoing HTTP proxy
+## Customizing the outgoing HTTP proxy
 
-Zulip supports routing all of its outgoing HTTP and HTTPS traffic
-through an HTTP `CONNECT` proxy, such as [Smokescreen][smokescreen];
-this includes outgoing webhooks, image and website previews, and
-mobile push notifications. You may wish to enable this feature to
-provide a consistent egress point, or enforce access control on URLs
-to prevent [SSRF][ssrf] against internal resources.
+To protect against [SSRF][ssrf], Zulip 5.0 and above default to
```

Comment:
Does specifying 5.0 here indicate you don't think we should backport this to 4.8?  

(It seems possible it'd be a merge conflict party, but haven't checked).

Reply:
It rebases essentially cleanly, with trivial conflicts for missing 391f78a9c18e920a0c7bfba1475c6ebab55f08b3 and df09607202306b25bc98499df2529d79d0364264.  And I do think this is a candidate to backport.

I was just a little wary marking that it was "since" a point release that doesn't exist yet, in case it makes folks go looking for it.  But I'll happily adjust to say 4.8.

Reply:
Yeah, I'd prefer to do that if we're going to backport this, since through merging this, we're surely going to put it in 4.8.